lib/index.js

```diff
@@ -18,6 +18,8 @@ for(const module of contextModules) {
   for(const c in contextConstants) {
     const contextUrl = contextConstants[c];
     bedrockConstants[c] = contextUrl;
-    jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
+    if(c.includes('_URL')) {
+      jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
+    }
   }
```

Comment:
This looks fragile and awkward.  The contexts are in their own map.  How about rewriting that whole loop as:
```js
  for(const c in contextConstants) {
    bedrockConstants[c] = contextConstants[c];
  }
  for(const [url, context] of contexts) {
    jsonLdDocumentLoader.addStatic(url, context);
  }
```

Reply:
:+1: seems like a decent idea.

Reply:
TY. Implemented.